src/database.py

```diff
@@ -106,21 +106,38 @@ def write_fraud_event_to_database(event, db_connection):
         with RunInTransaction(db_connection) as cursor:
             cursor.execute("""
                 INSERT INTO billing.fraud_events
-                (time_stamp, session_id, hashed_persistent_id, request_id, entity_id, fraud_event_id, fraud_indicator)
+                (
+                    time_stamp,
+                    session_id,
+                    hashed_persistent_id,
+                    request_id,
+                    entity_id,
+                    fraud_event_id,
+                    fraud_indicator,
+                    event_id,
```

Comment:
This might be silly, but seeing as it's the primary key, can we put `event_id` first in the list?

Reply:
Commit pushed!